### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by loido**

### CODE REVIEW

**Review by huydosgtech — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -115,26 +161,21 @@ Thanks goes to these wonderful people
...
+- [React Native Code Push](https://github.com/microsoft/react-native-code-push)
+- [React Native Fast Image](https://github.com/DylanVann/react-native-fast-image)
+- [React Native Gesture handle](https://github.com/software-mansion/react-native-gesture-handler)
+- [React Native Progress](https://github.com/oblador/react-native-progress)
+- [React Native Reanimated](https://github.com/software-mansion/react-native-reanimated)
+- [React Native Safe Area Context](https://github.com/th3rdwave/react-native-safe-area-context)
+- [React Native Screens](https://github.com/software-mansion/react-native-screens)
+- [React Redux](https://github.com/reduxjs/react-redux)
+- [React Saga](https://github.com/redux-saga/redux-saga)
+- [I18n JS](https://github.com/fnando/i18n-js)
```
Comment (honghoangsts) [inline README.md]: I don't think we need this section. I recall that we removed it in the project javascript

**Review by honghoangsts — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -25,21 +50,40 @@ $ npm -v && node -v
...
+    ```sh
+      npx react-native init YourProjectName --template rn-base-project-ts
+    ```
+2. Install node_modules
+   ```sh
+    yarn install
+   ```
+3. Run script install pod via package.json
+   ```sh
+    yarn postinstall
```
Comment (loc-nguyenthien) [inline README.md]: I don't think we need this, `yarn postinstall` is run automatically after we run `yarn install`

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (loido) [reply README.md]: oke updated

File: README.md (hunk 2 above)
Comment (loido) [reply README.md]: oke updated

File: README.md [hunk 3]
```diff
@@ -25,21 +50,35 @@ $ npm -v && node -v
...
+      npx react-native init YourProjectName --template rn-base-project-ts
+    ```
+2. Install node_modules
+   ```sh
+    yarn install
+   ```
+    > **Note**
+    > Pod install automatically after run `yarn install`  
+
+4. Run `Android` or `Ios` via package.json
```
Comment (loc-nguyenthien) [inline README.md]: Since you have removed step 3, change this to 3

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (loido) [reply README.md]: updated

**Review by honghoangsts — APPROVED** [review]

**Review by ngochuyduong — APPROVED** [review]

**Review by loc-nguyenthien — APPROVED** [review]

**Pull request merged**